src/cloud-api-adaptor/pkg/securecomms/sshproxy/sshproxy_test.go

```diff
@@ -136,32 +135,8 @@ func TestSshProxyWithNamespace(t *testing.T) {
 
 	clientSshPeer, serverSshPeer := getPeers(t)
 
-	namespace := uuid.NewString()
-	nsPath := "/run/netns/" + namespace
-	// Create a new network namespace
-	newns, err := netns.NewNamed(namespace)
-	if err != nil && !errors.Is(err, fs.ErrExist) {
-		if errors.Is(err, fs.ErrPermission) {
-			t.Skip("Skip due to missing permissions - run privileged!")
-		}
-		t.Errorf("netns.NewNamed(%s) returned err %s", namespace, err.Error())
-	}
-	defer func() {
-		newns.Close()
-		if err := netns.DeleteNamed(namespace); err != nil {
-			t.Errorf("failed to delete a named network namespace %s: %v", namespace, err)
-		}
-	}()
-
-	link, err := netlink.LinkByName("lo")
-	if err != nil {
-		t.Fatal(err)
-	}
-
-	// bring the interface up
-	if err := netlink.LinkSetUp(link); err != nil {
-		t.Fatal(err)
-	}
+	ns, namespace := tuntest.NewNamedNS(t, "test-TestSshProxyWithNamespace")
```

Comment:
Are these the right variable names as I assumed that `ns` was short for namespace?